Artifacts/windows-install-windows-updates/Artifactfile.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+  "title": "Installs the latest Windows Updates",
+  "description": "Installs the latest patches, security fixes and updates via the Windows Update PowerShell Module",
+  "publisher": "Microsoft",
+  "tags": [
+    "Windows"
+  ],
+  "targetOsType": "Windows",
+  "runCommand": {
+    "commandToExecute": "powershell.exe -ExecutionPolicy bypass \"& ./artifact.ps1\""
+  }
+}
```

Comment:
Use the `postDeployActions` instead to `restart` the VM.
```JSON
...
  "runCommand": {
  },
  "postDeployActions": [
    {
      "action": "restart"
    }
  ]
...
```

Reply:
Discussed offline - we'll leave the restart in the script but better comment on why...